lib/helpers.js

```diff
@@ -50,6 +50,11 @@ export const createRetry = (retries = RETRIES) => {
       .catch(error => {
         // debugger
         tries++
+
+        if (error.body && error.body.statusCode == 401) {
```

Comment:
This makes a lot of sense to me 👍 I'm also wondering if we should use triple equals instead? `===`?

Reply:
yep `===` seems to be more accurate 👍 since both of them are int
seems i had been lazy to click on third `=` 😆